include/rocksdb/options.h

```diff
@@ -375,6 +375,20 @@ struct DBOptions {
   // bottlenecked by RocksDB.
   DBOptions* IncreaseParallelism(int total_threads = 16);
 #endif  // ROCKSDB_LITE
+  // This specifies the traces file path.
+  std::string trace_path = "";
+
+  // This specifies the traces file name.
+  std::string trace_name_postfix = "";
+
+  // Max trace file size in byte
+  uint64_t max_trace_file_size = 1 * 1024 * 1024 * 1024;
```

Comment:
I suggest that you do `1ULL * 1024 * 1024 * 1024` or `static_cast<uint64_t>(1) * 1024 * 1024 * 10224`.
Should people in the future change the default to something like `4 * 1024 * 1024 * 1024`, this value will overflow.